_im-plugin/index-transforms/transforms-apis.md

```diff
@@ -28,6 +28,7 @@ PUT _plugins/_transform/<transform_id>
 {
   "transform": {
   "enabled": true,
+  "continuous": true,
```

Comment:
Could you add this in the same place in the sample response? It will be returned there as well as a confirmation to the user.

Reply:
Good catch. Added.